hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -124,6 +125,14 @@
       tags = ConfigTag.CLIENT)
   private boolean checksumVerify = true;
 
+  @Config(key = "checksum.combine.mode",
+      defaultValue = "MD5MD5CRC",
+      description = "The combined checksum type [MD5MD5CRC / COMPOSITE_CRC] "
+          + "determines which algorithm would be used to compute checksum for "
+          + "file checksum. Default checksum type is MD5MD5CRC.",
+      tags = ConfigTag.CLIENT)
+  private String checksumCombineMode = Options.ChecksumCombineMode.MD5MD5CRC.name();
```

Comment:
Please add `ozone.client.checksum.combine.mode` config to ozone-default.xml file.